bindings/python/google_benchmark/benchmark.cc

```diff
@@ -130,18 +130,25 @@ NB_MODULE(_benchmark, m) {
       .value("kAvgIterationsRate", Counter::Flags::kAvgIterationsRate)
       .value("kInvert", Counter::Flags::kInvert)
       .export_values()
-      .def(nb::self | nb::self);
+      .def("__or__", [](Counter::Flags a, Counter::Flags b) {
+        return static_cast<int>(a) | static_cast<int>(b);
+      });
 
   nb::enum_<Counter::OneK>(py_counter, "OneK")
       .value("kIs1000", Counter::OneK::kIs1000)
       .value("kIs1024", Counter::OneK::kIs1024)
       .export_values();
 
   py_counter
-      .def(nb::init<double, Counter::Flags, Counter::OneK>(),
-           nb::arg("value") = 0., nb::arg("flags") = Counter::kDefaults,
-           nb::arg("k") = Counter::kIs1000)
-      .def("__init__", ([](Counter *c, double value) { new (c) Counter(value); }))
+      .def(
+          "__init__",
+          [](Counter* c, double value, int flags, Counter::OneK oneK) {
+            new (c) Counter(value, static_cast<Counter::Flags>(flags), oneK);
```

Comment:
Won't this throw if `flags` is not a member of `Counter::Flags`?

Reply:
No. C++ doesn't care about this. enum flags are common in C++. It's only nanobind that checks membership.